quiche/src/lib.rs

```diff
@@ -4473,6 +4493,8 @@ impl<F: BufFactory> Connection<F> {
 
                 if push_frame_to_pkt!(b, frames, frame, left) {
                     self.blocked_limit = None;
+                    self.data_blocked_sent_count =
+                        self.data_blocked_sent_count.saturating_add(1);
```

Comment:
Note that this will count retransmisson

Reply:
Thanks for the heads up.  I think that at least for my purpose that's fine.